velox/functions/lib/MapFromEntries.cpp

```diff
@@ -30,9 +30,12 @@ static const char* kIndeterminateKeyErrorMessage =
     "map key cannot be indeterminate";
 static const char* kErrorMessageEntryNotNull = "map entry cannot be null";
 
-// See documentation at https://prestodb.io/docs/current/functions/map.html
 class MapFromEntriesFunction : public exec::VectorFunction {
  public:
+  // If throwOnNull is true, will return null if input array is null or has
+  // null entries (Spark's behavior), instead of throwing exceptions (Presto's
+  // behavior).
+  MapFromEntriesFunction(const bool throwOnNull) : throwOnNull_(throwOnNull) {}
```

Comment:
Comment suggestion:

```
@param throwOnNull If true, throws exception when input array is null or contains null entry. Otherwise, returns null.
```